lib/postgresql.js

```diff
@@ -119,7 +119,12 @@ PostgreSQL.prototype.executeSQL = function (sql, params, callback) {
       // console.log(err);
       if (log) log(sql, time);
       process.nextTick(done); // Release the pooled client
-      callback(err ? err : null, data ? data.rows : null);
+
+      var affectedCount = data && typeof data.rowCount === 'number' &&
+          !isNaN(data.rowCount) ? data.rowCount : 0;
+
+      callback(err ? err : null, data ? data.rows : null, {count:
+          affectedCount});
```

Comment:
Please pass the full `data` object, it should be the responsibility of individual DAO methods to decide what properties of the pg response should be passed on to the caller and how they should be transformed.

I also think you should return `undefined` instead of `0` if `rowCount` is not a valid number. That way you will signal that the database did not correctly report number of affected records. Your current implementation silently hides this problem, it will be extremely difficult to troubleshoot consequences of this.